### PR TITLE
Add documents for simulation

### PR DESCRIPTION
**Pull request by 200km**
## Overview
Add documents for simulation

## Issue
- NA

## Details
Add documents for simulation

##  Validation results
NA

## Scope of influence
NA

## Supplement
NA

## Note
NA

### CODE REVIEW

File: Specifications/Simulation/Spec_SampleCase.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+- `sample_case.cpp`
+  - Write detailed process of the override functions
+  - Constructor
+    - Usually, users just need to initialize the constructor of `SimulationCase`.
+  - Destructor
+    - Users need to `delete` your defined members `spacecraft` and `ground_station`.
+  - `InitializeTargetObjects` function
+    - You have to make instances of your `spacecraft` and `ground_station` class here.
+    - You also need to register log out put by `LogSetup` function.
```
Comment (seki-hiro) [inline Specifications/Simulation/Spec_SampleCase.md:33]:
```suggestion
    - You also need to register log output by `LogSetup` function.
```

File: Specifications/Simulation/Spec_SampleGroundStation.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - Write detailed process of the override functions
+  - Constructor
+    - Initialize `GroundStation` and call the `SampleGroundStation::Initialize` function.
+  - Destructor
+    - Delete components.
+  - `LogSetup` function
+    - Set log you need.
+  - `Update` function
+    - Call the `GroundStation::Update` function.
+    - Call the ground station component's `Update` functions
```
Comment (seki-hiro) [inline Specifications/Simulation/Spec_SampleGroundStation.md:35]:
```suggestion
    - Call the ground station component's `Update` functions.
```

**Review by seki-hiro — APPROVED** [review]

File: Specifications/Simulation/Spec_SampleGroundStation.md (hunk 2 above)
Comment (200km) [reply Specifications/Simulation/Spec_SampleGroundStation.md]: fixed

File: Specifications/Simulation/Spec_SampleCase.md (hunk 1 above)
Comment (200km) [reply Specifications/Simulation/Spec_SampleCase.md]: fixed

**Pull request merged**